tests/src/test/scala/io/circe/magnolia/configured/ConfiguredSemiautoDerivedSuite.scala

```diff
@@ -187,7 +188,7 @@ class ConfiguredSemiautoDerivedSuite extends CirceSuite with Inside {
 
   "Configuration#useDefaults" should "fail if key is missing and no default was provided for parameter" in {
     assert(parse("{}").flatMap(j => WithDefaultValue.decoder(j.hcursor)) == Left(
-        DecodingFailure("Attempt to decode value on failed cursor", List(CursorOp.DownField("required")))
+        DecodingFailure(MissingField, List(CursorOp.DownField("required")))
```

Comment:
error messages were changed in this PR: https://github.com/circe/circe/pull/1617

but it's hard to tell for me whether it's equivalent and expected failure